### PR TITLE
Core-Instance annotations

### PR DESCRIPTION
**Pull request by jasmingacic**
# Summary of this proposal

Adding annotations to core_instance opeator on create

## Asana task(s) link.

https://app.asana.com/0/1204316201844829/1206049531392534

## Notes for the reviewer

Needs https://github.com/calyptia/core-operator/pull/344

### CODE REVIEW

File: k8s/client.go [hunk 1]
```diff
@@ -632,6 +632,10 @@ func (client *Client) DeployCoreOperatorSync(ctx context.Context, coreCloudURL,
 			Name:  "HTTPS_PROXY",
 			Value: httpsProxy,
 		},
+		{
+			Name:  "ANNOTATINS",
```
Comment (niedbalski) [inline k8s/client.go:636]: typo?

File: cmd/coreinstance/create_core_instance_operator.go [hunk 2]
```diff
@@ -343,6 +344,7 @@ func newCmdCreateCoreInstanceOperator(config *cfg.Config, testClientSet kubernet
 	fs.StringVar(&environment, "environment", "", "Calyptia environment name")
 	fs.StringVar(&httpProxy, "http-proxy", "", "http proxy to use on this core instance")
 	fs.StringVar(&httpsProxy, "https-proxy", "", "http proxy to use on this core instance")
+	fs.StringVar(&annotations, "annotations", "", "Custom annotations for pipelines. Format should be 'annotation1=value1,annotation2=value2'")
```
Comment (patrick-stephens) [inline cmd/coreinstance/create_core_instance_operator.go:347]: Should we build in support for labels now as well as part of this PR for future proofing? It feels like adding a few lines will sort it.

File: k8s/client.go (hunk 1 above)
Comment (jasmingacic) [reply k8s/client.go:636]: indeed

File: cmd/coreinstance/create_core_instance_operator.go (hunk 2 above)
Comment (jasmingacic) [reply cmd/coreinstance/create_core_instance_operator.go:347]: let's try not to cram all we can think of into a single PR

**Review by niedbalski — APPROVED** [review]

**Pull request merged**